protoc-gen-openapiv2/options/openapiv2.proto

```diff
@@ -385,6 +385,7 @@ message Schema {
   // A free-form property to include an example of an instance for this schema in JSON.
   // This is copied verbatim to the output.
   string example = 6;
+  map<string, google.protobuf.Value> extensions = 7;
```

Comment:
Which section within the swagger spec does this map to: https://swagger.io/docs/specification/2-0/swagger-extensions/? Just want to make sure we're not adding these somewhere they don't belong.

Reply:
If I'm understanding correctly, this is used for [`MessageOptions`](https://github.com/grpc-ecosystem/grpc-gateway/blob/185126621f298395754503abc6460e5d9e1f62b9/protoc-gen-openapiv2/options/annotations.proto#L29) annotations and so could be either operation parameters or responses (see https://github.com/grpc-ecosystem/grpc-gateway/blob/185126621f298395754503abc6460e5d9e1f62b9/protoc-gen-openapiv2/options/openapiv2.proto#L230) in the swagger schema.

This allows for annotations like this, e.g.:

```
message AMessage {
  option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_schema) = {
    extensions: {
      key: "x-something-something";
      value {
        string_value: "yadda";
      }
    }
  }
};
```